main.js

```diff
@@ -66,17 +87,55 @@ function setAPPListeners () {
  * Opens new window with index.html(Jitsi Meet is loaded in iframe there).
  */
 function createJitsiMeetWindow () {
-    jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
-    jitsiMeetWindow.loadURL(indexURL);
+  jitsiMeetWindow = new BrowserWindow(jitsiMeetWindowOptions);
+  jitsiMeetWindow.loadURL(indexURL);
+  microWindow = new BrowserWindow(microWindowOptions);
+  microWindow.loadURL(microModeURL);
 
-    jitsiMeetWindow.webContents.on('new-window', function(event, url) {
-        event.preventDefault();
-        electron.shell.openExternal(url);
-    });
+  jitsiMeetWindow.webContents.on('new-window', function(event, url) {
+      event.preventDefault();
+      electron.shell.openExternal(url);
+  });
 
-    jitsiMeetWindow.on("closed", () => {
-        jitsiMeetWindow = null;
-    });
+  jitsiMeetWindow.on("closed", () => {
+      jitsiMeetWindow = null;
+      microWindow = null;
+  });
+
+  setIPCListeners();
+}
+
+/**
+ * Sets the ipc listeners for messages from renderer processes
+ */
+function setIPCListeners() {
+  // ipcMain.on('log', (event, data) => {
+  //   console.log(data);
+  // });
+  ipcMain.on('mainWindowPeerCreated', () => {
```

Comment:
i wonder if there might be a better format for these messages where the 'destination' (main window vs micro window) was contained in a field rather than in the message name itself?